layouts/learning-paths/single.html

```diff
@@ -0,0 +1,13 @@
+{{ define "main" }}
+  {{ partial "breadcrumbs.html" . }}
+  <article class="prose max-w-none dark:prose-invert">
+    <img src="{{ .Params.image }}" alt="{{ .Title }}" class="w-full rounded-lg object-cover object-bottom h-96">
```

Comment:
We currently prefer cropping along the bottom of the graphic, but I should try moving the graphics to assets and [crop](https://gohugo.io/content-management/image-processing/#crop) with the [smartcrop](https://github.com/muesli/smartcrop#smartcrop) anchor.

```suggestion
    <img src="{{ .Params.image }}" alt="{{ .Title }}" class="w-full rounded-lg object-cover h-96">
```

Reply:
meh, it's fine for now, won't bother with cropping since the hero is responsive